src/components/collection-progress-group/index.tsx

```diff
@@ -22,11 +25,37 @@ function CollectionProgressGroup({ collection }: { collection: Collection }) {
 	 * Parse the progress-related information we need from the progress records,
 	 * current collection slug, and list of tutorials in this collection.
 	 */
-	const { completedTutorialCount, isInProgress, tutorialCount, tutorialCta } =
+	const { completedTutorialCount, isCompleted, isInProgress, tutorialCount } =
 		useMemo(
-			() => parseCollectionProgress(progressData, tutorials, { id, slug }),
+			() =>
+				parseCollectionProgress(progressData, tutorials.length, { id, slug }),
 			[progressData, tutorials, id, slug]
 		)
+	/**
+	 * Parse the CTA we want to show for the "next" tutorial in the collection.
+	 * This "next" tutorial varies based on the progress state
+	 */
+	const tutorialCta = useMemo(
```

Comment:
We now get the `tutorialCta` separately, since we've split out `getNextTutorialCta` from `parseCollectionProgress`.